docs/docs/components/api/README.md

```diff
@@ -199,14 +200,16 @@ Async options allow users to select prop values that can be programmatically-gen
   prevContext,
+  query,
 }) {},
 ```
 
 | Property      | Type      | Required? | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | ------------- | --------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `options()`   | `method`  | optional  | Typically returns an array of values matching the prop type (e.g., `string`) or an array of object that define the `label` and `value` for each option. The `page` and `prevContext` input parameter names are reserved for pagination (see below).<br>&nbsp;<br>When using `prevContext` for pagination, it must return an object with an `options` array and a `context` object with a `nextPageToken` key. E.g., `{ options, context: { nextPageToken }, }` |
 | `page`        | `integer` | optional  | Returns a `0` indexed page number. For use with APIs that accept a numeric page number for pagination.                                                                                                                                                                                                                                                                                                                                                         |
-| `prevContext` | `string`  | optional  | Return a string representing the context for the previous `options` execution. For use with APIs that accept a token representing the last record for pagination.                                                                                                                                                                                                                                                                                              |
+| `prevContext` | `string`  | optional  | Returns a string representing the context for the previous `options` execution. For use with APIs that accept a token representing the last record for pagination.                                                                                                                                                                                                                                                                                              |
```

Comment:
I only added an `s` on this line, so that the parameter `returns` a string (the same way the previous lines are worded)